Node/inventario.js

```diff
@@ -0,0 +1,61 @@
+function producto(opciones){
+	var opciones = opciones || {};
+	this.nombre=options.nombre || comida;
+	this.cantidad=options.cantidad || 5;
+	this.decrecer=opciones.decrecer || 1;

```

Comment:
Intentar no utilizar el this. Y, comprobad mediante varios para garantizar su correcto funcionamiento.

Reply:
corregido